packages/esm-patient-tests-app/src/test-results/results-viewer/results-viewer.extension.test.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { type ConfigObject, getDefaultsFromConfigSchema, useConfig } from '@openmrs/esm-framework';
+import { configSchema } from '../../config-schema';
```

Comment:
```suggestion
import { getDefaultsFromConfigSchema, useConfig } from '@openmrs/esm-framework';
import {  type ConfigObject, configSchema } from '../../config-schema';
```

Let's use the `ConfigObject` from the Tests app config schema.